compiler/rustc_mir_build/src/lib.rs

```diff
@@ -17,6 +17,7 @@ extern crate rustc_middle;
 
```

Comment:
What is remaining for this to deny the migration lints?

Reply:
There's only more unsafe op lints (which I've just migrated rn, those were tricky and took some time bc I had to split a single lint into different lints for every unsafe op), and pattern lints